pyronear/datasets/openfire.py

```diff
@@ -32,16 +32,19 @@ class OpenFire(VisionDataset):
         threads (int, optional): If download is set to True, use this amount of threads
             for downloading the dataset.
         num_samples (int, optional): Number of samples to download (all by default)
+        img_folder (str or Path, optional): Location of image folder. Default: <root>/OpenFire/images
     """
 
     url = 'https://gist.githubusercontent.com/frgfm/f53b4f53a1b2dc3bb4f18c006a32ec0d/raw/c0351134e333710c6ce0c631af5198e109ed7a92/openfire_binary.json'
     training_file = 'training.pt'
     test_file = 'test.pt'
     classes = [False, True]
 
-    def __init__(self, root, train=True, download=False, threads=16, num_samples=None, **kwargs):
+    def __init__(self, root, train=True, download=False, threads=16, num_samples=None, img_folder=None, **kwargs):
```

Comment:
Would you mind going to next line after `num_samples` term please?
I'll open a PR with pep8 verification (including max_line_length=120) and this line is longer apparently